benchmarks/text_hybrid_scoreboard/sqlite_fts5_bench.py

```
@@ -0,0 +1,202 @@
+    build_seconds = time.perf_counter() - build_start
+
+    sql = "SELECT rowid, bm25(docs_fts, 3.0, 1.0) AS score FROM docs_fts WHERE docs_fts MATCH ? ORDER BY score LIMIT ?"
+    # Warm outside the timed loop and assert that the query shape has hits.
+    warm = list(conn.execute(sql, (args.query, args.top_k)))
+    if not warm:
+        raise SystemExit(f"query {args.query!r} returned no rows")
+
+    durations: list[int] = []
+    total_results = 0
+    search_start = time.perf_counter_ns()
+    for i in range(args.queries):
```

Ruff (0.15.15)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)